ccpp/data/GFS_typedefs.F90

```diff
@@ -1696,6 +1696,8 @@ module GFS_typedefs
 !     logical               :: land_iau_do_stcsmc_adjustment
 !     real(kind=kind_phys)  :: land_iau_min_T_increment
 
+    character(len=2)     :: sfc_file_version = 'V1'
```

Comment:
Please add definition comments for documentation.
character(len=2)     :: sfc_file_version = 'V1'     ! version number of input surface file
 
Also are you going to make a separate PR to add a namelist variable sfc_file_version in gfs_physics_nml? If possible, I think we can have the namelist change in this PR. So we will have a namelist default value and have model%sfc_file_version set up in control_initialiazation step.